cypress/support/commands/drSetRNG.js

```diff
@@ -0,0 +1,12 @@
+import s from '../../integration/dryRunner/selectors'
+
+const setRNG = mode => {
+  Cypress.log({
+    name: `RNG ${mode}`,
+    message: `Set RNG mode to ${mode}`,
+  })
+
+  return cy.get(s[`${mode}_RNG_RADIO`]).parent().click()
```

Comment:
I thought I posted this comment but I somehow cannot find it again. Here it is:

I like the way you think. I would have written it like this though:

```js
cy.get('[data-testid="RNG-input"]').find(`[value="${mode}"]`).check()
```

That would imply setting `[data-testid="RNG-input"]` to all radio inputs of course. :)